package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "altis-reusable-blocks",
-  "version": "1.0.0",
+  "version": "0.1.3",
```

Given this isn't published on npm we can swap this for the following, then the version doesn't need to be updated in this file:
```suggestion
  "private": true,
```

---

Love it @roborourke! I knew there was a `private` property that you could add to the `package.json` but I did **_not_** know that it removes the requirement of a version #.